src/panels/config/entities/entity-registry-settings.ts

```diff
@@ -1051,9 +1051,9 @@ export class EntityRegistrySettings extends SubscribeMixin(LitElement) {
         .buttons {
           box-sizing: border-box;
           display: flex;
-          padding: 0 24px 24px 24px;
+          padding: 8px 24px 24px 24px;
           justify-content: space-between;
-          padding-bottom: max(env(safe-area-inset-bottom), 24px);
+          padding-bottom: max(env(safe-area-inset-bottom), 16px);
```

Comment:
if you change this, you should also change the normal padding rule... but we shouldnt change this, as this is the same for every dialog